openpgp/packet/aead_encrypted.go

```diff
@@ -23,6 +23,9 @@ type AEADEncrypted struct {
 const aeadEncryptedVersion = 1
 
 func (ae *AEADEncrypted) parse(buf io.Reader) error {
+	if V5Disabled {
+		return errors.UnsupportedError("support for parsing v5 entities is disabled; change `config.V5Disabled` if needed")
```

Comment:
It might be more helpful to tell them to set the tag in this context, then they don't have to figure out how to set the flag:
```suggestion
		return errors.UnsupportedError("support for parsing v5 entities is disabled; build with `-tags v5` if needed")
```
(and same for the other errors)